### ISSUE TITLE
False error for D417 with multiline arguments

### ISSUE THREAD

**Issue by cazador481** [charliermarsh/ruff#1662]
I am getting the following error:
```
D417 Missing argument descriptions in the docstring: `endpoint\`, \`token\`
```
when running ruff with 
```toml
[tool.ruff.pydocstyle]
convention = 'google'
```
On the following code block:
```python
def init_otel(
    console: bool = False,
    default_provider: bool = True,
    service_name: Optional[str] = None,
    endpoint: Optional[str] = None,
    token: Optional[str] = None,
    version: Optional[str] = None,
    resource: Optional[resources.Resource] = None,
) -> Optional[TracerProvider]:
    """Initialize otel telemetry connections.

    Args:
        console: Send to console if true, otherwise send to endpoint
        service_name: Service name, picks up the default HWINFCI's service name
        version: service version. Will try to automatically pick this up.
        default_provider: Is this the default provider for OTEL
        endpoint:
            http otel endpoint
            Overridden with ENV[HWINFCI_OTEL_ENDPONT].  If set to console, will output to console
        resource: Additional resources to add
        token:
            OTEL authorization token
            Overridden with ENV[HWINFCI_OTEL_TOKEN]

    Returns:
        Optionally returns a TracerProvider
    """
```

When running through flake8 I don't get that error.

**Comment by charliermarsh** [comment]
(The reason this works in Flake8, confusingly, is that it's incorrectly inferring that it's a NumPy-style docstring.)

**Comment by charliermarsh** [comment]
Will fix this today, thanks.
